features/plugins.feature

```diff
@@ -31,7 +31,8 @@ Feature: Functionality of Importer and Exporter Plugins
         Given We use the config "basic_onefile.yaml"
         When We run "jrnl --version"
         Then the output should contain pyproject.toml version
-        And The output should contain "<plugin_name> : <version> from jrnl.<source>.<type>.<filename>" 
+        And the output should contain "<plugin_name> : <version> from jrnl.<source>.<type>.<filename>"
```

Comment:
It's good that we're fixing up the tests, so this is fine. For future test fixes/updates, though, can we focus on the `/tests/bdd/features` directory instead?